docs/builders/docker.mdx

```diff
@@ -585,7 +585,7 @@ above and example configuration properties are shown below:
 post-processors {
   post-processor "docker-tag" {
       repository = "12345.dkr.ecr.us-east-1.amazonaws.com/packer"
-      tag = ["0.7"]
+      tag = "0.7"
```

Comment:
@azr Oops, this was correct. just the `docker-import` was wrong.
Maybe we should use `tags = ["0.7"]` as that's what documented
https://www.packer.io/docs/post-processors/docker/docker-tag